processor/transformprocessor/internal/common/testhelper.go

```diff
@@ -0,0 +1,39 @@
+// Copyright  The OpenTelemetry Authors
```

Comment:
Is this meant only for use in tests? If so, can we call it something like `helper_test.go` so that it is excluded appropriately?

Reply:
Yes good idea.